stock_split_picking/models/stock.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+# Copyright 2013-2015 Camptocamp SA - Nicolas Bessi
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, models
+from odoo.exceptions import UserError
+
+
+class StockPicking(models.Model):
+    """Adds picking split without done state."""
+
+    _inherit = "stock.picking"
+
+    @api.multi
+    def split_process(self):
+        """Use to trigger the wizard from button with correct context"""
+        for pick in self:
+            if pick.state == 'draft':
+                raise UserError(_('Mark as todo this picking please.'))
+            if all([x.qty_done == 0.0 for x in pick.pack_operation_ids]):
+                raise UserError(
+                    _('You must enter done quantity in order to split your '
+                      'picking in several ones.'))
+            ctx = {
+                'active_model': self._name,
+                'active_ids': self.ids,
+                'active_id': len(self.ids) and self.ids[0] or False,
+                'do_only_split': True,
+                'default_picking_id': len(self.ids) and self.ids[0] or False,
+            }
+
+            view = self.env.ref('stock.view_backorder_confirmation')
+            return {
```

Comment:
Instead of showing the wizard, just create it on the background and call the same method as if you click on "Create backorder" button. For user security, add to the "Split" button a confirm attribute.